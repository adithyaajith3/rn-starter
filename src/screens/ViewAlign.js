import React from 'react';
import { StyleSheet, View, Text } from 'react-native';


const ViewAlign = () => {
    return <View style={ styles.container }>
        <View style={ styles.child }></View>
        <View style={ styles.childCenter }></View>
        <View style={ styles.child }></View>
    </View>
}

styles = StyleSheet.create({
    container: { 
        flexDirection: 'row',
        backgroundColor: '#fff',
        justifyContent: 'space-between',
        height: 100
    },
    child: {
        width: 50,
        backgroundColor: 'blue',
        height: 50,
    },
    childCenter: {
        alignSelf: "flex-end",
        backgroundColor: 'green',
        height: 50,
        width: 50
    }
})

export default ViewAlign;