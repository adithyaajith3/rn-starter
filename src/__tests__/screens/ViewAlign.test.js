import React from 'react';
import ViewAlign from '../../screens/ViewAlign';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const tree = renderer.create(<ViewAlign />).toJSON();
    expect(tree).toMatchSnapshot();
  });
  