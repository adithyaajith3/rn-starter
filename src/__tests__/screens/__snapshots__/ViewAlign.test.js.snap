// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#fff",
      "flexDirection": "row",
      "height": 100,
      "justifyContent": "space-between",
    }
  }
>
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "alignSelf": "flex-end",
        "backgroundColor": "green",
        "height": 50,
        "width": 50,
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "blue",
        "height": 50,
        "width": 50,
      }
    }
  />
</View>
`;
